hack/e2e/e2e-run.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+ROOT_DIR=$(dirname "${BASH_SOURCE[0]}")/../..
+SCRIPT_DIR=${ROOT_DIR}/hack/e2e
+CONFIG_DIR=${ROOT_DIR}/test/e2e/yaml
+CONFIG_FILENAME=e2e-config.yaml
```

Comment:
Please help to merge `e2e-config.yaml`, `e2e-tools.yaml` and `e2e-variables.yaml` as well. See #596 for more info. 

Also, please test this script locally to see if it works, and if it passes, please include the results logs or Screenshots in the Conversation.

Reply:
OK，I will check it.